frontend/.devcontainer/devcontainer.json

```
@@ -5,7 +5,8 @@
     "ghcr.io/devcontainers/features/common-utils:2.4.2": {
       "username": "none",
       "upgradePackages": false
-    }
+    },
+    "git-lfs": "latest"
```

Why do we need `lfs`?

---

Discussed offline, seems reasonable. Could we use the fully qualified syntax? I.e.:
```suggestion
    },
    "ghcr.io/devcontainers/features/git-lfs:1.2.3": {}
```